WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAccountPendingView.swift

```diff
@@ -0,0 +1,67 @@
+import SwiftUI
+
+struct InPersonPaymentsStripeAccountPending: View {
+    let deadline: Date?
+
+    var body: some View {
+          VStack {
+              Spacer()
+
+              VStack(alignment: .center, spacing: 42) {
+                  Text(Localization.title)
+                      .font(.headline)
+                  Image(uiImage: .paymentErrorImage)
+                      .resizable()
+                      .scaledToFit()
+                      .frame(height: 180.0)
+                  Text(message)
+                      .font(.callout)
+                  InPersonPaymentsSupportLink()
+              }
+              .multilineTextAlignment(.center)
+
+              Spacer()
+
+              InPersonPaymentsLearnMore()
+          }
+          .padding(24.0)
+      }
+
+    private var message: String {
+        guard let deadline = deadline else {
+            DDLogError("In-Person Payments not avilable. Stripe has pending requirements without known deadline")
+            return Localization.messageUnknownDeadline
+        }
+        return String(format: Localization.messageDeadline, deadline.toString(dateStyle: .medium, timeStyle: .none))
```

Comment:
I tested on both English and Spanish, and I think `.long` fits the sentence better

```suggestion
        return String(format: Localization.messageDeadline, deadline.toString(dateStyle: .long, timeStyle: .none))
```